kafkagen/genproto.go

```diff
@@ -53,6 +54,10 @@ var included = []string{
 	"TxnOffsetCommitResponse",
 	"EndTxnRequest",
 	"EndTxnResponse",
+	"CreateTopicsRequest",
```

Comment:
Also, in kafkaprotocol/protocol.go an entry needs to be added to SupportedApiVersions

Reply:
yes that is already present. Kindly check:
```
	{ApiKey: APIKeyCreateTopics, MinVersion: 0, MaxVersion: 5},
	{ApiKey: APIKeyDeleteTopics, MinVersion: 0, MaxVersion: 5},
```